platform/ios/src/MGLMapView_Private.h

```diff
@@ -52,8 +52,6 @@ FOUNDATION_EXTERN MGL_EXPORT MGLExceptionName const _Nonnull MGLUnderlyingMapUna
 /// Synchronously render a frame of the map.
 - (void)renderSync;
 
-- (nonnull mbgl::Map *)mbglMap;
```

Comment:
This seems orthogonal to other changes. Why this?

Reply:
Moved this to another PR.